.editorconfig

```diff
@@ -0,0 +1,9 @@
+# http://EditorConfig.org
+root = true
+
+[*.js]
+charset = utf-8
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
```

Comment:
Personally, I believe this is a good idea, but should be added in a separate PR where just this topic can be discussed. See #29

Reply:
Oops, yes of course